patch-package
--- a/node_modules/react-native-material-kit/iOS/RCTMaterialKit/MKTouchableManager.m
+++ b/node_modules/react-native-material-kit/iOS/RCTMaterialKit/MKTouchableManager.m
@@ -8,7 +8,7 @@
 
 #import <React/RCTViewManager.h>
 #import <React/RCTEventDispatcher.h>
-#import <UIView+React.h>
+#import <React/UIView+React.h>
 #import "MKTouchable.h"
 
 @interface MKTouchableManager : RCTViewManager <MKTouchableDelegate>
--- a/node_modules/react-native-material-kit/iOS/RCTMaterialKit/TickViewManager.m
+++ b/node_modules/react-native-material-kit/iOS/RCTMaterialKit/TickViewManager.m
@@ -7,7 +7,7 @@
 //
 
 #import <React/RCTViewManager.h>
-#import <UIView+React.h>
+#import <React/UIView+React.h>
 #import "TickView.h"
 
 @interface TickViewManager : RCTViewManager
